sections/Chapter 7 Sprint 5/planning.tex

```diff
@@ -14,11 +14,41 @@ \subsection{User stories}
 \userStory{the functionality layer}{query an RDF graph in a standardized way}{do not have to worry about changes to the database}
 
 \subsubsection*{Granulating the epics}
-After talking to the functionality layer, we found out details about how they want database access to work. 
+After talking to the functionality layer, we found out some of the details about how they want database access to work. 
+
 \begin{enumerate}
+    \item It should be possible to write RDF data to the Fuseki database using the Terse RDF Triple Language (Turtle) syntax and file format \cite{TurtleFormat}.
     \item They want to be able to send SPARQL queries directly to an endpoint to fetch the data.
     \item They would prefer if the response was in XML format. They have given us an example.
     \item They do not know the size of the data they will be querying.
     \item The route for GET request should have a string parameter called "query" containing the SPARQL query as a string.
     The query will be in URL-coded format.
 \end{enumerate}
+
+We have gotten enough insight to begin work on the database, however due to the other groups not knowing exactly what they want, the planning for this sprint will be a little ad hoc and unpredictable.  
+This means that there is a possibility of newly discovered works during the sprint.
+
+With this in mind, we first transferred the remaining report backlog from the last sprint and received three user stories from the PO. 
+From this backlog, we have formulated the goals for this sprint.
+
+From the epics, we created a backlog. As mentioned in section \ref{Whatwelearnedsprint3}, we decided to work with a different unit of measurement for determining the size of a task.
+Instead of using hours, we now simply interpret the points assigned to a task as what determines its size.
+
+We partook in a game of Inversion \cite{InversionThinking} where we tried to find problems with our ideas by attempting to see them from the point of view of a potential examiner. 
```

Comment:
```suggestion
We partook in a game of Inversion \cite{InversionThinking} where we tried to find issues with our prioitization by imagining what would happen if we did not work on the given tasks.
```

What is written is contradicted in the next sentence (line 38). Tried to fix so it better matches how I interpreted our game.

Reply:
"by imagining what would happen if we did not work on the given tasks." 
How about: "by imagining what would happen if we did not complete the given tasks."?